src/Router.js

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import Login from './pages/Minji/Login/Login';
-import Main from './pages/Minji/Main/Main';
 import Join from './pages/Minji/Main/Join';
-
+import DoneJoin from './pages/Minji/Main/DoneJoin';
+import Thread from './pages/Minji/Thread/Thread';
 const Router = () => {
   return (
     <BrowserRouter>
       <Routes>
         <Route path="/" element={<Login />} />
-        <Route path="/main" element={<Main />} />
-        <Route path="/join" element={<Join />} />
+        <Route path="/minji.join" element={<Join />} />
+        <Route path="/minji.donejoin" element={<DoneJoin />} />
+        <Route path="/minji.thread" element={<Thread />} />
```

Comment:
Route 컴포넌트의 path값에는 띄어쓰기 단위로  - 기호를 붙여주는게 웹표준 방식입니다. 
```suggestion
        <Route path="/minji-join" element={<Join />} />
        <Route path="/minji-donejoin" element={<DoneJoin />} />
        <Route path="/minji-thread" element={<Thread />} />
```